js/script.js

```diff
@@ -170,10 +170,38 @@ seeProjButtons.forEach((button, index) => {
 const errorMessage = document.querySelector('.error-message');
 const form = document.querySelector('.form');
 const email = document.querySelector('#email');
+const userName = document.querySelector('#name');
+const message = document.querySelector('#message');
+const formDatakey = 'formData';
+
+let formObject = {
+  userName: '',
+  email: '',
+  message: '',
+};
+
 form.addEventListener('submit', (event) => {
+  formObject.userName = userName.value;
+  formObject.email = email.value;
+  formObject.message = message.value;
+  localStorage.setItem(formDatakey, JSON.stringify(formObject));
+
```

Comment:
- [ ] It is a project requirement that **When the user changes the content of any input field, the data is saved to the local storage.** Your implementation only saves data to local storage when the form is submitted. Please apply onchange event listeners to the input fields so that they can save the change when the text within them change. The picture below shows te form fully filled but nothing is save in local storage because the form is not yet submitted. 🧇 
![local](https://github.com/Kccendana/portfolio_project/assets/99656687/44531d79-e22b-490d-8a67-4d68b45d84d6)